internal/node/controller/init.go

```diff
@@ -27,7 +27,7 @@ func generatePDSAppConfig(nodeConfig *config.NodeConfig) types.PostAppRequest {
 				Driver: "docker",
 				DriverConfig: map[string]interface{}{
 					"env": []string{
-						"PDS_HOSTNAME=ethangraf.com",
+						"PDS_HOSTNAME=localhost",
```

Comment:
what does this do if it's set to `localhost` but uses your private key below?

Reply:
The key below is to help with rotating certs, so has no effect on our current usage of the PDS. We will need to work out how to manage this properly in the long run, but not really high priority in my opinion right now.